financialaid/views.py

```diff
@@ -224,23 +227,56 @@ class FinancialAidActionView(UpdateAPIView):
     queryset = FinancialAid.objects.all()
 
 
-class GetLearnerPriceForCourseView(APIView):
+class CoursePriceListView(APIView):
     """
-    View for retrieving a leaner's price for a course run
+    View for retrieving a learner's price for course runs in all enrolled programs
+    """
+    authentication_classes = (SessionAuthentication,)
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request, *args, **kwargs):
+        """
+        GET handler
+        """
+        user = request.user
+        program_enrollments = ProgramEnrollment.objects\
```

Comment:
Can you put parentheses around the expression here instead of using `\` to allow multiple lines?